src/main/java/com/sap/oss/phosphor/fosstars/model/value/VersionRange.java

```diff
@@ -38,13 +36,21 @@ public VersionRange(
     this.versionEnd = versionEnd;
   }
 
-  @JsonGetter("versionStart")
-  public Optional<String> versionStart() {
+  /**
+   * Returns the start version.
+   *
+   * @return The start version if available.
+   */
+  public Optional<String> start() {
```

Comment:
I think this would have fixed it? https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/oss/phosphor/fosstars/model/subject/oss/MavenArtifact.java#L13